domain-models-runtime/src/main/java/org/folio/rest/persist/PgExceptionUtil.java

```diff
@@ -154,9 +154,12 @@ public static String getMessage(Throwable throwable) {
     }
     Map<Character, String> map = new HashMap<>();
     map.put('M', ((PgException) throwable).getErrorMessage());
+
     map.put('D', ((PgException) throwable).getDetail());
+
     map.put('S', ((PgException) throwable).getSeverity());
     map.put('C', ((PgException) throwable).getCode());
+
```

Comment:
Are these blank lines intended? If yes please explain why in this PR's description.

Reply:
No Julian! those blank lines are not intended , while cloning in IntelliJ IDE it reformats the code. i am new to the IntelliJ IDE thats why i was not aware about this. Anyway i have tried to reformat the code via IDE . Kindly review .